code/src/layouts/app-header/AppHeader.test.jsx

```diff
@@ -0,0 +1,18 @@
+import { render } from "@testing-library/react";
+import AppHeader from "@/layouts/app-header/AppHeader";
+import Header from "@/layouts/app-header/components/header/Header";
+import Categories from "@/layouts/app-header/components/categories/Categories";
+
+jest.mock("@/layouts/app-header/components/header/Header", () => () => (
+  <div>Header</div>
+));
+jest.mock("@/layouts/app-header/components/categories/Categories", () => () => (
+  <div>Categories</div>
+));

```

Comment:
Same here

Reply:
> Same here

See above the screenshot. 
It will be `Invalid variable access: Categories` in this case